monai/csrc/resample/pushpull_cpu.cpp

```diff
@@ -97,19 +97,25 @@ MONAI_NAMESPACE_DEVICE { // cpu
         bool do_sgrad)
         : dim(dim),
           bound0(bound.size() > 0 ? bound[0] : BoundType::Replicate),
-          bound1(bound.size() > 1 ? bound[1] : bound.size() > 0 ? bound[0] : BoundType::Replicate),
```

Comment:
The changes here are more than just `if`/`else`. With one of our other linters, we check line length <120, but I'm guessing that deepsource is going for <80? Might be worth configuring.

Reply:
this is from the recent clang-format upgrade, formatted some previously ignored style issues